sdk/state/open.go

```diff
@@ -47,7 +67,8 @@ func (c *Channel) OpenTxs(d OpenAgreementDetails) (txClose, txDecl, formation *t
 		IterationNumber:            1,
 		AmountToInitiator:          0,
 		AmountToResponder:          0,
-		Asset:                      d.Asset,
+		// TODO - change to use all assets, simplifying for now
```

Comment:
only using the first asset for this PR, will change to use all of them in the next PR